nomos-services/data-availability/verifier/src/storage/adapters/rocksdb.rs

```diff
@@ -15,14 +14,15 @@ use overwatch_rs::{
     services::{relay::OutboundRelay, ServiceData},
     DynError,
 };
-// internal
-use crate::storage::DaStorageAdapter;
+// std
```

Comment:
Nit: throughout the codebase we try to group modules in order of std, crates and internal (with respective comments). Unfortunately we don't have a linter for that.

Reply:
Neither can be auto fmt iirc. Maybe @ntn-x2 have some magic trick related to that?

Reply:
I think the formatter will maintain relative order if the different groups are spaced with a new line. Honestly I don't see the benefit of adding the comments to signal the different sections. Once we add a new line, any new import will automatically be added to the right section. [This fmt rule](https://rust-lang.github.io/rustfmt/?version=v1.4.38&search=#StdExternalCrate%5C%3A) is what we could try with. I think that would already be a big step ahead without the need for these custom solution. If you guys agree, I could open a new PR with that and perhaps a few other useful formatting rules, like perhaps [this one](https://rust-lang.github.io/rustfmt/?version=v1.4.38&search=#Crate%5C%3A).

Reply:
Yeah, those rules may work I think.  wdyt guys? @bacv @AlejandroCabeza @youngjoon-lee @andrussal

Reply:
Sorry I saw the links did not point to the exact section I was referring to. The first link is for the `StdExternalCrate` rule, while the second is for the `imports_granularity = "Crate"` setting.

Reply:
> Sorry I saw the links did not point to the exact section I was referring to. The first link is for the `StdExternalCrate` rule, while the second is for the `imports_granularity = "Crate"` setting.

Yeah, I just searched them manually from the link hints :sweat_smile:

Reply:
IIRC A couple months ago we talked about this exact options (`StdExternalCrate` and  `crate`) being ideal but still in nightly or beta. If they are released then let's go.
If they're not, removing the comments is still a good idea as that simplifies adoption of those options when they are released.

Reply:
You are right, it seems they are still no nightly. Nevertheless, I don't see how that would be a blocker? We can use a nightly toolchain in the CI for the fmt job, and there's way to also configure local machines to run a nightly version as the default formatter. We've done that elsewhere in the past. Would you guys be ok with that? I don't think using nightly for formatting has any security or important implications whatsoever.